core/src/test/java/org/jboss/modcluster/mcmp/ResetRequestSourceTestCase.java

```diff
@@ -49,7 +51,17 @@ public class ResetRequestSourceTestCase {
     private final BalancerConfiguration balancerConfig = mock(BalancerConfiguration.class);
     private final MCMPRequestFactory requestFactory = mock(MCMPRequestFactory.class);
 
-    private ResetRequestSource source = new ResetRequestSourceImpl(this.nodeConfig, this.balancerConfig, this.requestFactory);
+    private final ResetRequestSource source = new ResetRequestSourceImpl(this.nodeConfig, this.balancerConfig, this.requestFactory);
+
+    private final MCMPRequest configRequest = mock(MCMPRequest.class);
+    private final MCMPRequest enableRequest = mock(MCMPRequest.class);
+    private final MCMPRequest disableRequest = mock(MCMPRequest.class);
+    private final MCMPRequest stopRequest = mock(MCMPRequest.class);
+
+    private final static String JVM_ROUTE = "host1";
+    private final static String HOST_NAME = "host";
+    private final static String ALIAS1 = "alias1";
+    private final static String ALIAS2 = "alias2";
```

Comment:
Error:  /home/runner/work/mod_cluster/mod_cluster/core/src/test/java/org/jboss/modcluster/mcmp/ResetRequestSourceTestCase.java:61:19: 'static' modifier out of order with the JLS suggestions. [ModifierOrder]

Checkstyle hates the order here ;-)

Reply:
Should be fixed now.

Reply:
@TomasHofman Nope, still isn't (the JLS proposes "private static final String" order.

Reply:
Ha, sorry, forgot `git add .` :)

Reply:
So, it should be fixed NOW :)

Reply:
Who needs `git add` when you have `git commit -a` ;-)